lib/diagnostic.rb

```diff
@@ -100,13 +103,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.delete_at(-1, -2)
 
 # What would `arr` equal after:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
```

Comment:
Adding 99 to the index at `arr.length +  2` would give you `[12, 34, 56, 67, nil, nil, 99]`